webapp/src/containers/components/modals/styles.css

```diff
@@ -0,0 +1,4 @@
+.show {
+    display: block;
```

Comment:
`div` by default has `block` level, are we overriding anything here? if not can remove it